Documentation/freebsd-instructions.md

```diff
@@ -0,0 +1,200 @@
+Build CoreCLR on FreeBSD
+======================
+
+This guide will walk you through building CoreCLR on FreeBSD and running Hello World.  We'll start by showing how to set up your environment from scratch.
+
+Environment
+===========
+
+These instructions are written assuming the 10.1-RELEASE, since that's the release the team uses.
```

Comment:
This is a nit, but it would be nice if it said "assuming the FreeBSD 10.1-RELEASE" just to make it abundantly clear.